File: Software_Technologies/17_Exam_Prep/Cat_Shop/C#/CatShop/Models/Cat.cs
namespace CatShop.Models
{
    using System.ComponentModel.DataAnnotations;

    public class Cat
    {
        [Key]
        public int Id { get; set; }

        public string Name { get; set; }

        public string Nickname { get; set; }

        public double Price { get; set; }
    }
}

File: Software Technologies/17_Exam_Prep/Cat_Shop/JavaScript/models/cat.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    let Cat = sequelize.define("Cat",{
        name:{
            type: Sequelize.TEXT,
            allowNull: false,
        },
        nickname:{
            type: Sequelize.TEXT,
            allowNull: false,
        },
        price:{
            type: Sequelize.DOUBLE,
            allowNull: false,
        }
    },{
        timestamps:false
    });

    return Cat;
};